test/unit/StakingPool/calculatePremium.js

```diff
@@ -0,0 +1,480 @@
+const { expect } = require('chai');
+const { ethers } = require('hardhat');
+const { getCurrentTrancheId, calculateBasePrice, calculateSurgePremium, calculatePriceBump } = require('./helpers');
+const { mineNextBlock, setNextBlockTime } = require('../../utils/evm');
+const { daysToSeconds } = require('../../../lib/helpers');
+const { AddressZero, MaxUint256, WeiPerEther } = ethers.constants;
+const { parseEther } = ethers.utils;
+const { BigNumber } = ethers;
+
+describe('calculatePremium', function () {
+  const periodInDays = 91.25;
```

Comment:
Is there any particular reason for choosing this period? A tranche has 91 days exactly if that's what the aim was.

Reply:
Made it a multiple of 365 days to make calculating the `premium` from `premiumForYear` easier.